torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -715,6 +713,12 @@ PjRtComputationClient::ExecuteReplicated(
                   ->Execute(std::move(argument_handles), execute_options,
                             returned_futures)
                   .value();
+
+    (*returned_futures)[0].OnReady(
+        std::move([timed, op = std::move(op)](xla::Status unused) mutable {
```

Comment:
Think the callback `OnReady` moves its argument implicitly, so no need for `std::move` on the input lambda.

Reply:
Yeah, you're right. I was following [this documentation for absl::AnyInvocable](https://github.com/abseil/abseil-cpp/blob/master/absl/functional/any_invocable.h#L83-L86), but that is only relevent if we assign the lambda to a variable.